core/assets/bundles/bundle_pt_BR.properties

```diff
@@ -157,37 +157,37 @@ save.import = Importar save
 save.newslot = Nome do save:
 save.rename = Renomear
 save.rename.text = Novo jogo:
-selectslot = Selecione um slot para salvar.
+selectslot = Selecione um lugar para salvar.
 slot = [accent]Slot {0}
-editmessage = Edit Message
+editmessage = Editar Mensagem
 save.corrupted = [accent]Arquivo corrompido ou inválido!
 empty = <vazio>
 on = Ligado
 off = Desligado
 save.autosave = Autosalvar: {0}
 save.map = Mapa: {0}
 save.wave = Horda {0}
-save.mode = Gamemode: {0}
+save.mode = Modo de jogo: {0}
 save.date = Último salvamento: {0}
 save.playtime = Tempo De Jogo: {0}
 warning = Aviso.
 confirm = Confirmar
 delete = Excluir
-view.workshop = View In Workshop
+view.workshop = Ver na Workshop
 ok = OK
 open = Abrir
-customize = Customize
+customize = Customizar
 cancel = Cancelar
 openlink = Abrir Link
 copylink = Copiar link
 back = Voltar
 data.export = Exportar Data
 data.import = Importar Data
 data.exported = Data exportada.
-data.invalid = Isso não é daa de jogo válida.
+data.invalid = Isso não é data de jogo válida.
 data.import.confirm = Importal data externa irá deletar[scarlet] toda[] sua data atual.\n[accent]Isso não pode ser desfeito![]\n\nQuando sua data é importada, seu jogo ira sair imediatamente.
 classic.export = Exportar data classica
-classic.export.text = [accent]Mindustry[] has just had a major update.\nClassic (v3.5 build 40) save or map data has been detected. Would you like to export these saves to your phone's home folder, for use in the Mindustry Classic app?
+classic.export.text = [accent]Mindustry[] Acabou de ter uma atualização.\nClassic (v3.5 build 40) Jogos salvos ou mapas foram detectados. Voce quer exportar estes jogos para o seu celular, para usar no Mindustry classico?
```

Comment:
This part doesn't make sense at all.
```
Mindustry Acabou de ter uma atualização.
Classic (v3.5 build 40) Jogos salvos ou mapas foram detectados [...]
```
I suggest editing it to
```
Mindustry acabou de ter uma atualização.
Foram detectados jogos ou mapas da versão Classic (v3.5 build 40). Você quer exportar esses jogos para a pasta principal do seu celular para usar no Mindustry Classic?
```
And there's more strings that doesn't make sense inside the game or aren't accented correctly. pls fix it